GHzDACs/Cleanup/dac.py

```diff
@@ -125,70 +183,84 @@ def connect(self, name, group, de, port, board, build):
         p.timeout(self.timeout)
         p.listen()
         yield p.send()
-        
+
         # Get board specific information about this device.
         # We talk to the labrad system using a new context and close it when
         # done.
         reg = self.cxn.registry
         ctxt = reg.context()
         p = reg.packet()
-        p.cd(['','Servers','GHz FPGAs'])
-        p.get('dac'+self.devName.split(' ')[-1], key='boardParams')
+        p.cd(['', 'Servers', 'GHz FPGAs'])
+        p.get('dac' + self.devName.split(' ')[-1], key='boardParams')
         try:
             resp = yield p.send()
             boardParams = resp['boardParams']
             self.parseBoardParameters(boardParams)
         finally:
             yield self.cxn.manager.expire_context(reg.ID, context=ctxt)
-    
+
     @inlineCallbacks
     def shutdown(self):
         """Called when this device is to be shutdown."""
         yield self.cxn.manager.expire_context(self.server.ID,
                                               context=self.ctx)
-    
+
+    def runSram(self, dataIn, loop, blockDelay):
+        raise NotImplementedError()
+
     # Register byte methods
-    
+
     @classmethod
     def regPing(cls):
         """Returns a numpy array of register bytes to ping DAC register"""
         regs = np.zeros(cls.REG_PACKET_LEN, dtype='<u1')
-        regs[0] = 0 # No sequence start
-        regs[1] = 1 # Readback after 2us
-        return regs 
-    
+        regs[0] = 0  # No sequence start
+        regs[1] = 1  # Readback after 2us
+        regs[51] = MONITOR_0
+        regs[52] = MONITOR_1
+        return regs
+
     @classmethod
     def regPllQuery(cls):
         """Returns a numpy array of register bytes to query PLL status"""
         regs = np.zeros(cls.REG_PACKET_LEN, dtype='<u1')
-        regs[0] = 1 # No sequence start
-        regs[1] = 1 # Readback after 2us
+        regs[0] = 0  # No sequence start
+        regs[1] = 1  # Readback after 2us
```

Comment:
I'm not exactly sure why the diff is showing a change here, because I think it should be 0 and I don't remember changing it. Here are the deets:

* Old boards
 * 0 = no start -- use for pings
 * 1 = start -- use for regular runs
 * 2, 3, 4 = various test functions
* new boards
 * 0 = idle
 * 1 = master
 * 2 = test
 * 3 = slave
 * always use 1 unless you are an idle or a slave board in bg.run. if you want to do a ping/whatever, set num reps to 0.